recipes/sol2/3.x.x/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp sol_test.hpp)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+
+enable_testing()
```

Comment:
You don't need to use ctest, because you are not running any test library. But' okay, it doesn't harm.

Reply:
if its ok I would like to keep it, the test runs super fast.

But what do you mean with not running any test library?
This recipe it makes use of the lua library, does this count?

Reply:
I mean, you are not using test cases in your code. It's just a code which consumes the package, not an unit test.

Reply:
it does the same as a unit test, if the checks are not as expected, it returns an error (number) 
unit tests do nothing different, they add just some more output text for the checks